policyengine_us/parameters/gov/states/mn/tax/income/credits/marriage/minimum_individual_income.yaml

```diff
@@ -1,13 +1,18 @@
 description: Minnesota provides a marriage credit for joint filers where the lesser income of the two spouses is at least this minimum amount.
 values:
   2021-01-01: 26_000
+  2023-01-01: 28_000
 
 metadata:
   period: year
   unit: currency-USD
   label: Minnesota marriage credit minimum individual income for eligibility
   reference:
+  - title: Minnesota Statutes 290.0675 MARRIAGE PENALTY CREDIT. Subd. 3.Credit amount.
+    href: https://www.revisor.mn.gov/statutes/cite/290.0675#stat.290.0675.3
```

Comment:
this doesn't have 26,000 or 28,000?

Reply:
seems not... I only found the rules regarding $2000 bracket limit there. And didn't see another tax code on marrige credits.

Reply:
can you look a bit more for connected laws? do not include references that do not support the parameter value

Reply:
I see, will search

Reply:
This is what the legal code states: We read it as "The number is published in the tax forms and cannot be raised by more than 2,000 in one year"

![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/ed9255e9-d4a9-4ed8-b947-c547ba51e598)

Reply:
Ok could you note this in a comment?